frontend/assets/stylesheets/utilities/_util-visibility.scss

```diff
@@ -71,6 +72,12 @@
     }
 }
 
+.desktop-only {
+    @include mq($until: tablet) {
```

Comment:
I'm not knowledgeable about scss, so I'm a bit confused here - you're creating a new style called `desktop-only`, but it says `$until: tablet` - can you explain what's going on here? How is it `desktop-only`?